scripts/schedule-release.js

```diff
@@ -0,0 +1,86 @@
+const inquirer = require('inquirer');
+const moment = require('moment');
+const {
+  isVersionValid,
+  isReleaseDateValid,
+  getVersionFromReleaseType,
+  setVersion,
+  setReleaseDate,
+} = require('./set-version');
+
+const questions = [
+  {
+    type: 'list',
+    name: 'changeType',
+    message: 'Select the type of the release.',
+    choices: [
+      'Major',
+      'Minor',
+      'Patch',
+      'Custom',
+    ],
+    filter: value => value.toLowerCase(),
+  },
+  {
+    type: 'input',
+    name: 'customVersion',
+    message: 'Enter the custom version number.',
+    when: answers => answers.changeType === 'custom',
+    validate: (value) => {
+      if (isVersionValid(value)) {
+        return true;
+      }
+
+      return 'The provided version is not a proper semver version number.';
+    },
+  },
+  {
+    type: 'input',
+    name: 'releaseDate',
+    message: 'Enter the release date in a form of DD/MM/YYY.',
+    validate: (value) => {
+      if (isReleaseDateValid(value)) {
+        return true;
+      }
+
+      return 'The provided date is invalid';
```

Comment:
It's a minor issue but can be helpful in avoiding irritating the release master while scheduling the release (especially if that person releases the package the first time). I mean it's worth adding logic that checks if the date isn't passed as past value and then the script should print specified error, not common "invalid".